doc/releases/changelog-dev.md

```diff
@@ -148,6 +148,22 @@
          [ 0.162,  0.477]])
   ```
 
+* Added the possibility to compute general Renyi entropies in the `ClassicalShadow` class.
+  [(#2959)](https://github.com/PennyLaneAI/pennylane/pull/2959)
+  We can access the general Renyi-$\alpha$ entropy of a given subsystem by specifying its `wires`.
+
+  >>> shadow = ClassicalShadow(bits, recipes)
+  >>> Renyi_entropy = shadow.entropy(wires=[0, 3], alpha=1.5)
```

Comment:
This needs to be wrapped in a code block for it to render correctly (same thing below).